frontend/packages/data-portal/app/components/Viewer/steps.tsx

```diff
@@ -52,212 +74,172 @@ export const proxyStepSelectors: { target: string; className: string }[] = [
   },
 ]
 
-export const getTutorialSteps: () => Step[] = () => [
-  {
-    target: '.neuroglancerIframe',
-    placement: 'center',
-    disableBeacon: true,
-    title: 'Exploring CryoET Data in Neuroglancer',
-    content: (
-      <StepContent variant="default">
-        <div className="p-6 bg-[#EBEBEB] rounded relative overflow-hidden min-h-[247px]">
-          <div className="absolute left-6 right-6 rounded overflow-hidden">
+function TransTour(props: I18nProps) {
+  return (
+    <I18n
+      components={{
+        span: <span />,
+        p: <p />,
+        div: <div />,
+        ul: <ul />,
+        li: <li />,
+        InfoIcon: <InfoIcon />,
+        doclink: <a href={NEUROGLANCER_DOC_LINK}>tmp</a>,
+      }}
+      {...props}
+    />
+  )
+}
+
+export const getTutorialSteps: (t: Translator) => Step[] = (t) => {
+  return [
+    {
+      target: '#neuroglancerIframe',
+      placement: 'center',
+      disableBeacon: true,
+      title: t('neuroglancerWT.main.title'),
+      content: (
+        <StepContent variant="default">
+          <div className="p-6 bg-[#EBEBEB] rounded relative overflow-hidden min-h-[247px]">
+            <div className="absolute left-6 right-6 rounded overflow-hidden">
+              <img
+                src="/images/neuroglancer_tour/viewer_example.png"
+                alt="Portal viewer example"
+                className="w-full rounded"
+              />
+            </div>
+          </div>
+          <p className="flex flex-col gap-10">
+            <TransTour i18nKey="neuroglancerWT.main.content" />
+          </p>
+        </StepContent>
+      ),
+    },
+    {
+      target: '.joyride-proxy-layer-group-viewer',
+      title: t('neuroglancerWT.step1.title'),
+      placement: 'left-start',
+      disableBeacon: true,
+      content: (
+        <StepContent variant="compact">
+          <TransTour i18nKey="neuroglancerWT.step1.content" />
```

Comment:
Something is slightly off with the formatting on this one in the list
<img width="379" height="366" alt="image" src="https://github.com/user-attachments/assets/de6fe287-65cd-49d5-a6eb-898b4fd3d4b5" />

Compare to the old
<img width="393" height="372" alt="image" src="https://github.com/user-attachments/assets/41aa02a9-3d08-4509-bc05-962159207c2b" />